packages/neuron-wallet/src/exceptions/address.ts

```diff
@@ -1,27 +1,34 @@
-import i18n from 'locales/i18n'
+import { t } from 'locales/i18n'
```

Comment:
I would suggest make this variable bit more expressive. Maybe `import {t as translate} from 'locals/i18n'` is a good alternative?

Reply:
> I would suggest make this variable bit more expressive. Maybe `import {t as translate} from 'locals/i18n'` is a good alternative?

The `t` is the standard name of this function, and its type is `TFunction` so I think it's clear enough for i18n users.

Reply:
🤔 seems that I don't need to export it from `locals/i18n`, it can be imported from `i18next` directly.

Reply:
If `i18next` can be used in all the places where `locals/i18n` is used, then I would suggest we should only use `i18next`. Using both `i18n` and `i18next` might be confusing at some point.

Reply:
I'm not sure why the `i18n` is exported from `locals/i18n`.

AFAIK the `locals/i18n` is just used to set up configuration and will take effect by imported in the entry. And then we can use `i18n` exported from `i18next` in other places. So I think `import i18n from 'locals/i18n'` can be replaced by `import i18n from 'i18next'` globally.

I'll check it later.

Reply:
Sounds good